shutup.css

```diff
@@ -649,7 +649,12 @@ a[href*="#respond"],
 a[href*="#comments"],
 
 /* computerbase.de */
-.article__comments-link,
+span.article__meta-li:has(a[class="article__comments-link js-thread-link"]),
+div.article-view__meta-right:has(a[class="article__comments-link article-view__comments-link-1 js-thread-link"]),
+.article-view__comments-link-2.js-thread-link,
+
+/* golem.de */
+.share-item.comments
```

Comment:
@stonerl FYI: This line is missing a comma at the end, so the parser is going to combine this selector with the one below it. I'll merge the update into the browser extension and fix this on my end. Thanks for your contribution!